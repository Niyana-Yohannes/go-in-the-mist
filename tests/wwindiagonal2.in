place S1
place A1
place S2
place B2
place S3
place C3
place S4
place D4
place A2
place E5
