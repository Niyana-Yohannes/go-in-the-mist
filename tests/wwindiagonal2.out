White wins!
S1A1S2B2S3C3S4D4A2E5
Thank you for playing!
